components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIGatewayManager.java

```diff
@@ -74,11 +74,12 @@ public synchronized static APIGatewayManager getInstance() {
 	 */
 	public void publishToGateway(WebApp api, APITemplateBuilder builder, String tenantDomain)
 	                                                                                      throws Exception {
+		APIIdentifier webAppId = api.getId();
```

Comment:
Formatting issues and Java doc comment issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files